drivers/platform/maxim/maxim_rtc.c

```diff
@@ -199,3 +199,16 @@ int32_t no_os_rtc_set_cnt(struct no_os_rtc_desc *dev, uint32_t tmr_cnt)
 
 	return 0;
 }
+
+/**
+ * @brief Set the current count for the real time clock.
+ * @param dev - The RTC descriptor.
+ * @param irq_time - The number of seconds at which the interrupt must occur.
```

Comment:
is this an interval ? or is it the number of seconds that should pass from the function call to the interrupt ?